fix for issue where null pointer exception is thrown when attribute value is not provided and matchType is regexp (#1400)
microsoft/ApplicationInsights-Java: CHANGELOG.md

# CHANGELOG

# Version 3.0.1 GA

> * fix for an issue where null pointer exception is being thrown when attribute value is not provided and matchType is regexp

Reviewer: ```suggestion
* Telemetry processor config throws null pointer exception when attribute value is not provided and matchType is regexp
```
